README.md

```diff
@@ -695,10 +695,10 @@ Translations of the guide are available in the following languages:
   num = 1234
   ```
 
-* <a name="rdoc-conventions"></a>
-    Use [Rdoc][rdoc] and its conventions for API documentation.  Don't put an
+* <a name="api-documentation"></a>
+    Use [YARD][yard] and its conventions for API documentation.  Don't put an
```

Comment:
Probably we can remove the second sentence here, as it seems unrelated to me.

Reply:
Ok. I'll remove the second sentence.